etc/.jshintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "bitwise": true,
+  "curly": true,
+  "eqeqeq": true,
+  "freeze": true,
+  "futurehostile": true,
+  "latedef": "nofunc",
+  "noarg": true,
+  "nonew": true,
+  "shadow": false,
+  "strict": false,
+  "undef": false
```

Comment:
Rather than attempt to enumerate all the predefined/global variables (e.g. `require`, `describe`, `beforeEach`, etc.) that are/will be used by the test-helper module and others in the directory, explicitly allow "undefined" variables here.